pages/papers/active-record-migrations-history-of-creation.mdx

```diff
@@ -4,36 +4,32 @@ lang: en
+**Audience Level:** _Beginer, All_
 
 ### Elevator Pitch (300 characters)
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You have 300 characters to sell your talk. This is known as the "elevator pitch". Make it as exciting and enticing as possible.
 </div>
 
-Active Record Migrations es lo que mas me impacto de Ruby on Rails, como ayuda esto a versionar nuestra estructura de base de datos es lo mejor, para demostrar lo sencillo y hermoso que es escribir migraciones, me anime a contarles una historia con migraciones.
+Active Record Migrations is what most impact me Ruby on Rails, how this helps to version our database structure is the best, to show how simple and beautiful it is to write migrations, I encourage you to tell a story with migrations.
 
 ### Description
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You should make the description of your talk as compelling and exciting as possible. Remember, you're selling both the organizers of the events to select your talk, as well as trying to convince attendees your talk is the one they should see.
 </div>
 
-Las migraciones son una forma conveniente de alterar nuestro esquema de base de datos de manera consistente y facil. Esto hace que no tengamos que escribir SQL a mano, lo que permite que nuestro esquema y nuestros cambios permanescan independientes.
+Migrations are a convenient way to alter our database schema consistently and easily. This means that we do not have to write SQL by hand, which allows our schema and our changes to remain independent.
 
-Para interiorizar bien los conceptos de migraciones sobre como crear, leer, actualizar y eliminar, una tabla de nuestra base de datos, les contare una historia en el cual repasaremos estos conceptos.
+To internalize well the concepts of migrations on how to create, read, update and eliminate, a table from our database, I will tell you the history of the creation according to the biblical chapter geminis 1:27 using migrations.
 
 ### Notes
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
  This is where you should explain things such as technical requirements, why you're the best person to speak on this subject, etc...
 </div>
 
-Soy estudiante en Codeable un bootcamp de fullstack web developer donde construimos cosas increibles usando ruby, react y mas. Estoy emocionado por contarles una historia narrada con migraciones, cómo esto ayuda al proceso de aprendizaje, como facilita a interiorizar conceptos y dejar que nuestra imaginacion vuele inspirnado a mas personas a escribir historias usando conceptos de programacion.
+I am a student at Codeable, a bootcamp of fullstack web developer where we build amazing things using ruby, react and more. I am excited to tell you a story told with migrations, how this helps the learning process, how it facilitates to internalize concepts and let our imagination fly inspiring more people to write stories using programming concepts.
```

Comment:
developers* Ruby and React with capital letters.

Reply:
tell you a story using** migrations.
let our imagination fly, inspiring more people to write stories ...

Reply:
This is completely optional: ... where we build amazing apps similar to Slack, Netflix or Trivago.